src/test/java/marquez/db/models/DbModelGenerator.java

```diff
@@ -15,14 +15,7 @@
 package marquez.db.models;
 
 import static java.util.stream.Collectors.toList;
-import static marquez.common.models.CommonModelGenerator.newConnectionUrl;
-import static marquez.common.models.CommonModelGenerator.newDatasetName;
-import static marquez.common.models.CommonModelGenerator.newDatasetUrn;
-import static marquez.common.models.CommonModelGenerator.newDatasourceName;
-import static marquez.common.models.CommonModelGenerator.newDatasourceUrn;
-import static marquez.common.models.CommonModelGenerator.newDescription;
-import static marquez.common.models.CommonModelGenerator.newNamespaceName;
-import static marquez.common.models.CommonModelGenerator.newOwnerName;
+import static marquez.common.models.CommonModelGenerator.*;
```

Comment:
@wslulciuc, what's our recommended best practice on star imports? I've seen it where if there are more than 6 imports from a library, we default to `*`, for example, but it varies team by team.

Reply:
Removed *. This was generate by default from the IDE. Can I know the default number so that I can set it in my IDE.